graphql/resolvers.go

```diff
@@ -2809,6 +2809,14 @@ func (r *childPatchResolver) TaskCount(ctx context.Context, obj *restModel.Child
 	return &taskCount, nil
 }
 
+func (r *childPatchResolver) BaseVersionID(ctx context.Context, obj *restModel.ChildPatch) (*string, error) {
+	baseVersion, err := model.VersionFindOne(model.BaseVersionByProjectIdAndRevision(*obj.Project, *obj.Githash).Project(bson.M{model.VersionIdentifierKey: 1}))
```

Comment:
VersionIdentifierKey is actually the projectId for the version, not the ID (poor naming). This still works for you because mongo still projects Id by default, but VersionIdKey is the correct key for the Id

Reply:
(just a comment, not a blocking review; I'll leave this one for Chaya)

Reply:
VersionIdentifierKey is being used in the patchResolver.BaseVersionID as well, so if VersionIdKey is more accurate, we should change it there as well. Thanks Annie!